Sources/Filters+Strings.swift

```diff
@@ -194,7 +194,7 @@ extension Filters.Strings {
   ///   - arguments: the arguments to the function; expecting zero
   /// - Returns: the string with first letter being uppercased
   /// - Throws: FilterError.invalidInputType if the value parameter isn't a string
-  private static func upperFirstLetter(_ value: String) -> String {
+  internal static func upperFirstLetter(_ value: String) -> String {
```

Comment:
The code style on the rest of the code base seem to be not mentioning access control if it's `internal` (the default); better be consistent here then?

Reply:
This is needed to access the method from the SwiftIdentifier bit of code.